scripts/publish_modules.sh

```diff
@@ -19,43 +19,13 @@
 set -e
 
 WORKING_DIR="$PWD"
-MAIN="main"
-
-checkout_branch()
-{
-	echo "Checkout $1"
-	git checkout $1
-	git fetch origin
-	git pull
-}
-
-commit_changes()
-{
-	pushd $WORKING_DIR
-	git config --global user.email "elyra-canvas@users.noreply.github.com"
-	git config --global user.name "Automated build"
-	git add ./canvas_modules/common-canvas/package.json
-	git status
-	git commit -m "$2"
-	echo "Push changes to $1"
-	git push
-	popd
-}
-
-setup_git_branch()
-{
-		git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
-		git fetch
-}
-
-setup_git_branch
-checkout_branch ${MAIN}
 
 cd ./canvas_modules/common-canvas
-npm version minor
+release=$(echo $GITHUB_REF | cut -d'v' -f2)
```

Comment:
This works but would GITHUB_REF_NAME be better?  Probably doesn't matter though.

Reply:
Changed it, prob better just in case we run into some weird case